benefits/static/css/styles.css

```diff
@@ -14,6 +14,11 @@ html {
   scroll-padding-top: 101px; /* Header Height (80px) + Skip Nav Height (21px) = 101px */
 }
 
+/* reCAPTCHA: Hide the badge */
+.grecaptcha-badge {
+  visibility: hidden;
```

Comment:
Just a question: why not `display: none;` instead?

Reply:
Honestly, I am not sure. It's what the Google reCAPTCHA docs FAQ recommends: https://developers.google.com/recaptcha/docs/faq#id-like-to-hide-the-recaptcha-badge.-what-is-allowed But I would _prefer_ `display: none;` so it's not on the DOM and doesn't take up any space.

Reply:
Let's do that

Reply:
Done: https://github.com/cal-itp/benefits/pull/2586/commits/b21f8060b17ae006aba526dcf42e0771519cdc85